recipe/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "eval-type-backport" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz"
+  sha256: f0576b4cf01ebb5bd358d02314d31846af5e07678387486e2c798af0e7d849c1
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - pip
+    - python
+    - wheel
```

```suggestion
    - setuptools
    - setuptools-scm
    - wheel
```
See https://github.com/alexmojaki/eval_type_backport/blob/v0.2.2/pyproject.toml#L2C36-L2C50